bin/utils/convert-argv.js

```diff
@@ -128,28 +128,32 @@ module.exports = function(...args) {
 	}
 
 	if (!configFileLoaded) {
-		return processConfiguredOptions({});
+		return processConfiguredOptions(null);
 	} else if (options.length === 1) {
 		return processConfiguredOptions(options[0]);
 	} else {
 		return processConfiguredOptions(options);
 	}
 
 	function processConfiguredOptions(options) {
-		const webpackConfigurationValidationErrors = validateSchema(webpackConfigurationSchema, options);
-		if (webpackConfigurationValidationErrors.length) {
-			const error = new WebpackOptionsValidationError(webpackConfigurationValidationErrors);
-			console.error(error.message, `\nReceived: ${typeof options} : ${JSON.stringify(options, null, 2)}`);
-			process.exit(-1); // eslint-disable-line
+		if (options) {
+			const webpackConfigurationValidationErrors = validateSchema(webpackConfigurationSchema, options);
+			if (webpackConfigurationValidationErrors.length) {
+				const error = new WebpackOptionsValidationError(webpackConfigurationValidationErrors);
+				console.error(error.message, `\nReceived: ${typeof options} : ${JSON.stringify(options, null, 2)}`);
+				process.exit(-1); // eslint-disable-line
+			}
+		} else {
+			options = {};
 		}
 
 		// process Promise
-		if (typeof options.then === "function") {
+		if (options && typeof options.then === "function") {
```

Comment:
Now that you have the if/else, `options` will never be undefined right? So this additional check is not needed

Reply:
checks for `options.then` object prop so will throw. We're setting props on the object later these checks, I think we "assumed" we either found or did not find the props before validating args that aren't protected by 0CJS (entry and output)

Reply:
I'm not talking about .then, I'm talking about `options &&`. It's redundant. You already assigned it before with `options = {}`. 

So or you keep this check and remove the assignment or keep the assignment and remove the check.

Reply:
Thanks for pointing it out, changed now!